ansible/roles/software/ckan/inventory/tasks/additional-tasks.yml

```diff
@@ -36,3 +40,38 @@
     path: /usr/lib/datapusher/lib/python2.7/site-packages/ckanserviceprovider/web.pyc
     state: absent
   notify: restart apache2
+
+- name: "set up cron job for beaker cache cleanup"
+  block:
+    - name: install beaker cache cleanup script
+      template: src=beaker-cache-cleanup.sh.j2 dest=/usr/local/bin/beaker-cache-cleanup.sh owner=root group=root mode=0755
+
+    - name: install pgpass file for beaker-cache-cleanup
+      copy:
+        content: |
+          {{ inventory_ckan_db_host }}:5432:{{ inventory_db_name }}:{{ inventory_db_user }}:{{ inventory_db_pass }}
+        # TODO should be an app user variable
+        dest: /root/.pgpass
+        mode: "0600"
+        owner: root
+        group: root
+
+    - name: schedule beaker-cache-cleanup cron
+      cron:
+        name: "clean beaker cache in db"
+        cron_file: ckan
+        job: /usr/local/bin/beaker-cache-cleanup.sh
```

Comment:
Instead of running the script directly here, we should run a supervisor job from cron. That way supervisor will handle things like logging, retries, and serialization of jobs.